dashboard/app/models/pd/workshop.rb

```diff
@@ -81,6 +81,7 @@ class Pd::Workshop < ActiveRecord::Base
       SUBJECT_CS_IN_S_PHASE_3_SEMESTER_2 = 'Phase 3 - Semester 2'
     ],
     COURSE_CSP => [
+      SUBJECT_CSP_SUMMER_WORKSHOP = 'Summer Workshop',
```

Comment:
Well, I'm not sure either :) That is my assumption, but it could be the other way around. I just want to make sure we follow up and adjust if necessary.